src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -678,6 +678,7 @@ def load_arguments(self, _):
                    help='Enable terminate notification')
         c.argument('ultra_ssd_enabled', ultra_ssd_enabled_type)
         c.argument('scale_in_policy', scale_in_policy_type)
+        c.argument('force_deletion', arg_type=get_three_state_flag(), is_preview=True, help='This property allow you to specify if virtual machines chosen for removal have to be force deleted when a virtual machine scale set is being scaled-in.')
```

Comment:
`action='store_true'` should be a more appropriate option than `get_three_state_flag()`. Moreover, other similar parameters `force_update` also use `action='store_true'`, they should be consistent